### PR TITLE
Send detailed error message when installed standalone simulator failed

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: src/common/utility.ts [hunk 1]
```diff
@@ -632,7 +632,7 @@ export class Utility {
         let state: DockerState;
         let errorMsg: any;
         try {
-            await Executor.executeCMD(outputChannel, "docker", { shell: true }, "version");
+            await Executor.executeCMD(outputChannel, "docker2", { shell: true }, "version");
```
Comment (adashen) [inline src/common/utility.ts]: docker2?

File: src/common/InstallResult.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (adashen) [inline src/common/InstallResult.ts]: this file is only related to simulator right? How about to edge folder?

File: src/edge/simulator.ts [hunk 3]
```diff
@@ -281,9 +282,7 @@ export class Simulator {
             req.on("response",  (res) => {
                 if (res.statusCode === 200) {
                     req.pipe(unzipper.Extract({ path: Simulator.WindowsStandaloneSimulatorFolder }))
-                    .on("close", () => resolve()).on("error", (e) => reject(e));
-                } else if (res.statusCode === 404) {
```
Comment (adashen) [inline src/edge/simulator.ts]: 404 is not specially handled?

File: src/edge/simulator.ts [hunk 4]
```diff
@@ -201,6 +194,14 @@ export class Simulator {
         });
     }
 
+    private sendTelemetry(eventName: string, errMsg: string = null) {
```
Comment (adashen) [inline src/edge/simulator.ts]: what is the benefit of wrapping this method? same way to call telemetryclient?

File: src/common/InstallResult.ts (hunk 2 above)
Comment (SLdragon) [reply src/common/InstallResult.ts]: OK

File: src/edge/simulator.ts (hunk 3 above)
Comment (SLdragon) [reply src/edge/simulator.ts]: Because use cannot set the version or download url manually, , so we do not need to special handle the 404 case

**Pull request merged**